Dockerfile

```diff
@@ -44,12 +44,22 @@ RUN python3.9 -m venv $VIRTUAL_ENV
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
 
+#Install smart-phase
+RUN curl -sSL --retry 10 https://github.com/paulhager/smart-phase/archive/refs/tags/v1.2.0.tar.gz > sphase.tar.gz \
+    && tar -zxf sphase.tar.gz
```

Comment:
Prefer:

```
RUN curl -sSL --retry 10 https://github.com/paulhager/smart-phase/archive/refs/tags/v1.2.0.tar.gz | tar zx
```

No point saving the file if you are doing a composite command to unpack.